crates/shrs_core/src/plugin.rs

```diff
@@ -11,12 +11,15 @@ use crate::{
 pub struct PluginMeta {
     pub name: String,
     pub description: String,
+    pub help: String,
 }
 impl PluginMeta {
-    pub fn new<S: ToString>(name: S, description: S) -> Self {
+    pub fn new<S: ToString>(name: S, description: S, help: Option<S>) -> Self {
+        let help_string = help.map_or(String::from("no help text"), |h| h.to_string());
```

Comment:
hm instead of having the fallback here, let's make the `meta.help` field itself an option, and do the` help.map_or ...` stuff when we actually display the help message.

Reply:
Refactored.